packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/CompleteProfile/index.tsx

```diff
@@ -246,6 +246,7 @@ class CompleteProfile extends PureComponent<Props, State> {
                 <LinkItem
                   onClick={this.startVerification}
                   isComplete={isVerifiedId}
+                  isPending={isIdPending}
```

Comment:
same confusion here.  could we also pass the full prop names along to the child component?